src/isWilsonPrime.js

```diff
@@ -20,6 +20,14 @@
  */
 function isWilsonPrime(num) {
   // write code here
+  let factNum = num < 2 ? 1 : num - 1;
+    if (factNum === 0 || factNum === 1)
+      factNum = 1;
+    for (var i = factNum - 1; i >= 1; i--) {
+      factNum *= i;
+    }
+  return Number.isInteger((factNum + 1) / (num * num));
```

Comment:
add new line before return
```suggestion

  return Number.isInteger((factNum + 1) / (num * num));
```